tests/e2e/api_event_receiver_group_test.go

```diff
@@ -0,0 +1,234 @@
+package e2e
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/graph-gophers/graphql-go"
+	"github.com/sassoftware/event-provenance-registry/tests/common"
+	"gotest.tools/v3/assert"
+)
+
+func TestCreateAndGetGroup(t *testing.T) {
+	client := common.NewHTTPClient()
+
+	tests := map[string]struct {
+		input          eventReceiverGroupInput
+		receiverInputs []eventReceiverInput
+	}{
+		"group with no receivers": {
+			input: eventReceiverGroupInput{
```

Comment:
This should fail. I would not expect to be able to create a group without a receiver. There's no point.

Reply:
That scenario is covered in #70 . I moved this test case to be checked for failure, this was a leftover from when I wrote tests to pass rather than intentionally failing if we had bugs / missing functionality.